### PR TITLE
Resolve meta asset urls in additional meta assets

### PR DESCRIPTION
**Pull request by therouv**
### Description
It is currently not possible to additional meta assets (e.g. a msapplication-TileImage) to the <head> as the image path of this assets are not resolved, since this meta value uses the `content` attribute instead of `src` or `href` and is used to specify the path for the tile images for Windows devices.

This pull requests fixes this behavior and resolves the url for the defined meta assets. The meta key `msapplication-TileImage` was added by default since it is a very common meta asset.

### Fixed Issues (if relevant)
1. magento/magento2#5023: It is not possible to add MS tile image meta via default_head_blocks.xml

### Manual testing scenarios
1. Open `default_head_blocks.xml` of your theme.
2. Add additional meta asset image, e.g. *msapplication-TileImage* like this: `<meta name="msapplication-TileImage" content="images/favicons/default/mstile-144x144.png"/>`
3. If you inspect the rendered html code of the generated page, the asset url will now be resolved and the image will be found.

### Contribution checklist
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/magento2?pullRequest=16187) <br/>All committers have signed the CLA.

**Comment by magento-engcom-team** [comment]
Hi @therouv. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on Pull Request changes
- `@magento-engcom-team give me new test instance` - deploy **NEW** test instance based on Pull Request changes
- `@magento-engcom-team give me {$VERSION} instance` - deploy Vanilla Magento instance for Issue or Pull Request

For more details, please, review the [Magento Contributor Assistant](https://github.com/magento/magento2/wiki/Magento-Contributor-Assistant) documentation

**Review by orlangur — CHANGES_REQUESTED** [review]
Nowadays all changes must be applied to `2.3-develop` first (see https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html#rules). Please prepare a new pull request, this one should be `on hold` until the latter is merged.

**Comment by orlangur** [comment]
Closing due to inactivity. Feel free to reach me out anytime later if you wish to continue work on this pull request and it will be reopened.
Thanks for collaboration!

**Pull request closed without merging**
